maquetado/scss/book-list.scss

```diff
@@ -0,0 +1,37 @@
+.book-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  max-height: 768px;
+  max-width: 1366px;
+  padding: 88px 215px 8px 215px;
+
+  .book-item-list {
+    align-items: flex-start;
+    box-shadow: 4px 4px 10px $black-russian;
+    display: flex;
+    flex-direction: column;
+    margin: 18px;
+    max-height: 300px;
+    max-width: 198px;
+  
+    .book-image-mini{
+      max-height: 240px;
+      max-width: 198px;
+      padding: 28px 28px 12px 28px;
+    }
+  
+    .book-title-small{
+      color: $black;
+      font-size: $font-small;
+      font-weight: bold;
+      padding: 0 28px 8px 28px;
+    }
+  
+    .book-span-small{
+      color: $grey;
+      font-size: $font-very-small;
+      padding: 0 28px 23px 28px;
```

Comment:
The padding should be in book-item-list: `padding: 28px;`.
book-image-mini should has margin-bottom: 12px instead of padding
book-title-small should has margin-bottom: 8px instead of padding.
book-span-small shouldn't has margin and padding

Reply:
Done! That was another me, someone of the past who not understand the real concept behind margins and padding :sweat_smile:

Reply:
But you now understand, so it is more important 😄